src/main.js

```diff
@@ -281,12 +281,13 @@ define('peaks', [
       get: function () {
         var self = this;
 
-        function addSegment (startTime, endTime, editable, color, labelText) {
+        function addSegment (startTime, endTime, editable, color, labelText, id) {
```

Comment:
Done! That'd be great if you're happy to take a look at #6, otherwise I can try to find time.